Owncloud iOs Client/Files/Edit/EditFileViewController.h

```diff
@@ -19,14 +19,22 @@
 
 #define shareMainViewNibName @"EditFileViewController"
 
+extern NSString *iPhoneDoneEditFileTextMessageNotification;
```

Comment:
Consider using a delegate instead of a notification if only one class needs to be informed about changes in EditFileVC